docker-compose.yml

```diff
@@ -0,0 +1,9 @@
+version: '2'
+
+services:
+  jekyll:
+    image: bretfisher/jekyll-serve
```

Comment:
Does Jekyll not have an official image?

Reply:
it does, but this image runs `bundle install` and then `jekyll serve`, which is desirable behaviour and saves us writing a `Dockerfile` which does exactly that?